### DISCUSSION TITLE
Superscript/Subscript markup?

### DISCUSSION THREAD

**Discussion by tmarplatt** [vkbo/novelWriter#1330] [Ideas]
I don't mean to allow for feature creep but have those two formattings been considered for inclusion? Subscript and superscript text are not exactly uncommon, although there's the possibility of being creative with unicode for most use cases. But that kind of resourcefulness kind of eludes less technical users.

(By the way the global hotkey Ctrl+Shift+U does not work for entering an unicode character in novelWriter, is that a bug? I'm on Linux Mint 20 Cinnamon. Copying and pasting unicode works fine.)

**Comment by vkbo** [comment]
Sub/superscript has been considered, but the issue is syntax. There is nothing in makrdown for it, so something will need to be made up or adapted from another markup style. The solution in markdown is to use html codes, but novelWriter does not allow for inline html.

As for the `Ctrl+Shift+U` command, it is not blocked by novelWriter in any way, and it works on my system. I run Debian 11 with Gnome. The compose key feature also works fine if it's set up and enabled.

**Reply by vkbo** [reply]
I've looked at the extended markdown listed on [markdownguide.org](https://www.markdownguide.org/extended-syntax/), and perhaps those are good solutions. I've added the strikethrough from there, and highlight too is a candidate.

The mentioned syntax for superscript is `H~2~O` for "H<sub>2</sub>O" and `X^2^` for "X<sup>2</sup>". I believe that is simple and clear enough to not clutter the editor too much.

The proposed syntax for footnotes is also the one I plan to implement with a slight modification.
